### PR TITLE
speed displayed in km/h

### PR DESCRIPTION
**Pull request by gusciuag**
⛓ **Description:**

- Helps #50 

The car now continually sends speed information which is then displayed in the app in km/h format. The speedometer icon added to the screen in order to clearly indicate what the number refers to.

🛠 **Type of change:**
- [x] The application displays the speed in km/h
- [x] The number is truncated to 2 decimals
- [x] The screen displays speedometer icon

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

File: arduino/smartcar/smartcar.ino [hunk 1]
```diff
@@ -78,36 +76,55 @@ void setup() {
...
+        }
+
 void loop() {
  
-  obstacle();
- 
   if (mqtt.connected()) {
     mqtt.loop();
     const auto currentTime = millis();
+    mqtt.publish(speedometer, String(car.getSpeed()));
```
Comment (gusandchbh) [inline arduino/smartcar/smartcar.ino:127]: I think this line should be removed. This functionality is implemented on line 141.

File: arduino/smartcar/smartcar.ino [hunk 2]
```diff
@@ -15,23 +15,22 @@ WiFiClient net;
 //empty for local host connection
 const char ssid[] = " ";
 const char pass[] = " ";
- 
- 
+const String speedometer = "car/status/odometer/speed";
```
Comment (gusandchbh) [inline arduino/smartcar/smartcar.ino:18]: I think this line should be removed too. Or changed to "/smartcar/speedometer" and then use it in line 141.

**Review by gusandchbh — COMMENTED** [review]
Before the app recieves information about the speed of the car it shows the text "TextView", I think this should be changed to "0.0 km/h". 

There are some conflicts that need to be resolved and also this pull request is missing the DoD checklist.

**Review by gusandchbh — APPROVED** [review]
Code works as intended.

**Review by gusandchbh — APPROVED** [review]

**Review by Adrianmeda — APPROVED** [review]

**Pull request merged**